src/main/java/TaskControl.java

```diff
@@ -0,0 +1,101 @@
+public class TaskControl {
+    private final Task[] tasks;
+    private int taskCount;
+
+    public TaskControl() {
+        int MAX_TASKS = 100;
+        this.tasks = new Task[MAX_TASKS];
+        taskCount = 0;
+    }
+
+    public int getTaskCount() {
+        return taskCount;
+    }
+
+    public void printList() {
+        if (taskCount == 0) {
+            System.out.println("I have not been given any tasks! Start by creating an event, todo or deadline :)");
```

Comment:
You can probably rewrite this without the else (making the happy path prominent, just put a return here?)